lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -229,6 +229,12 @@
             - "I(True) enable menu to select boot device, I(False) to disable it. By default is chosen by oVirt/RHV engine."
         type: bool
         version_added: "2.5"
+    bios_type:
+        description:
+            - "Sets bios type, necessary for some operating systems and secure boot.
```

Comment:
Maybe metion from which verion of ovirt is which type available

Reply:
Good idea, looks like [4.3](http://ovirt.github.io/ovirt-engine-api-model/4.3/#types/bios) as omitted in [4.2](http://ovirt.github.io/ovirt-engine-api-model/4.2/#types/bios) according to the documentation.  done